### DISCUSSION TITLE
How to use this in aws lambda

### DISCUSSION THREAD

**Discussion by kishaningithub** [psycopg/psycopg2#1410] [Q&A] [answered]
Given aws lambda does not have libpq installed. what is the recommended way to use this library to use this in aws lambda?

**Comment by dvarrazzo** [comment, marked as answer]
I assume the psycopg2-binary package works, doesn't it?

**Reply by kishaningithub** [reply]
Is that the recommended way? I did not try it because the disclaimer said don't use it it production, that's why I had this question too

**Reply by dvarrazzo** [reply]
The alternative of using psycopg2-binary is to install the libpq. If you choose to run your software on a platform you don't have control of it's not really a problem we can solve.

However the current binary packages, shipping OpenSSL 1.1.x, have proven stable enough and not as problematic as the versions used when that disclaimer warning was written. I think you can use it no problem.

**Reply by szymic1** [reply]
@dvarrazzo Do you think about publishing ARM binary package? It can be interesting for AWS Lamda users (https://aws.amazon.com/blogs/aws/aws-lambda-functions-powered-by-aws-graviton2-processor-run-your-functions-on-arm-and-get-up-to-34-better-price-performance/)

**Reply by dvarrazzo** [reply]
I think we do that already.
